src/controllers/auth.js

```diff
@@ -19,9 +19,15 @@ const saveUserToTokenAndCookie = (req, res) => {
     maxAge: 1000 * 60 * 60 * 24 * 14,
   });
 
-  res.redirect('/');
+  res.redirect('/api/auth/me');
+};
+
+const protectedEndpoint = (req, res) => {
```

Comment:
As I said this code block is not necessary, and a little comment. I think you should check if there is a req.user. And then send the user as json